Item/Item.module.less

```
@@ -24,6 +24,22 @@
 		}
 	}
 
+	.slotAfter,
+	.slotBefore {
+		align-items: center;
+
+		// Remove the specific edge margin on direct children so the slot can define the proper
+		// spacing for itself.
+
+		& > div:first-child {
+			margin-inline-start: 0;
+		}
+
+		& > div:last-child {
+			margin-inline-end: 0;
+		}
+	}
```

To sync the code with sandstone, 

```suggestion
	.slotAfter,
	.slotBefore {
		line-height: 0;  // Attempt to get the children of this to always vertically-align their middles with the text

		> * {
			// vertically align all direct children
			vertical-align: middle;

			// Remove the specific edge margin on direct children so the slot can define the proper
			// spacing for itself.
			&:first-child {
				margin-inline-start: 0;
			}
			&:last-child {
				margin-inline-end: 0;
			}
		}
	}
```